src/components/styled-components.tsx

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components';
+
+export const StyledInput = styled.input`
+  border: 1px gray #777777;
+`;
+
+export const StyledLabel = styled.label`
+  font-size: 12px;
+  font-weight: regular;
+  color: gray;
+  margin-top: 12px;
+  display: flex;
+  flex-direction: column;
+`;
+
+export const Button = styled.button`
```

Comment:
só pra manter o padrão que vc fez. o que acha?
```suggestion
export const StyledButton = styled.button`
```